src/js/tabs/configuration.js

```diff
@@ -908,9 +908,7 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         $('._smallAngle').hide();
         if(semver.gte(CONFIG.apiVersion, "1.37.0")) {
             $('input[id="configurationSmallAngle"]').val(ARMING_CONFIG.small_angle);
-            if (SENSOR_CONFIG.acc_hardware !== 1) {
-              $('._smallAngle').show();
-            }
+            $('.accelNeeded').toggle(SENSOR_CONFIG.acc_hardware !== 1);
```

Comment:
Usually adding a `change();` to the end of the change method is enough and in this way you only hide it in one place. Have you tested it.